packages/flutter/lib/src/cupertino/bottom_tab_bar.dart

```diff
@@ -56,6 +56,7 @@ class CupertinoTabBar extends StatelessWidget implements PreferredSizeWidget {
     this.currentIndex = 0,
     this.backgroundColor,
     this.activeColor,
+    this.activeBackgroundColor,
```

Comment:
We may want to assert that this is non-null.

Reply:
its optional, but i added the correct default color (transparent) like in the Android variant